README.md

```diff
@@ -230,7 +230,7 @@ discontinued. For documentation on the older command options, refer to [README.m
 
 Client certificates are an alternative to traditional authentication mechanisms. These allow their users to make authenticated requests to a server, using a public certificate, and an optional private key that verifies certificate ownership. In some cases, the private key may also be protected by a secret passphrase, providing an additional layer of authentication security.
 
-Newman supports SSL client certificates, via the following CLI options:
+Newman supports SSL client certificates, via the following CLI options (available with Newman `v3` style `run` only):
```

Comment:
No point stressing this factor all over. It's becoming a burden and cautionary. Instead let's make the CLI help more eloquent.